third_party/autograph/test/impl/api_test.py

```diff
@@ -65,7 +60,20 @@ def __init__(self):
     self.x = 3
 
 
-class ApiTest(test.TestCase):
+def custom_constant(val):
+  return np.array(val)
+class ApiTest(unittest.TestCase):
+
+  def evaluate(self, x):
+    return x
+
+
+  def assertAllEqual(self, a, b):
+    if isinstance(a, np.ndarray) or isinstance(b, np.ndarray):
+      same = np.array(a) == np.array(b)
+      self.assertTrue(np.all(same))
+    else:
+      self.assertEqual(a, b)
```

Comment:
The blank lines need some moving around here. :D

Reply:
You are right, I am working on a followup to enable linting and tidy it up. I will do it in a separate PR.